platform-server-rest/src/main/java/uno/cod/platform/server/rest/controller/ChallengeController.java

```diff
@@ -21,17 +22,20 @@ public ChallengeController(ChallengeService service) {
     }
 
     @RequestMapping(value = RestUrls.CHALLENGES, method = RequestMethod.POST)
+    @PreAuthorize("isAuthenticated() and @securityService.isOrganizationAdmin(principal, #dto.organizationId)")
```

Comment:
i'm unsure if that should be limited to organization admins but we can change that afterwards